app/lib/fixAddress.js

```diff
@@ -0,0 +1,31 @@
+const fields = ['line1', 'line2', 'line3', 'city', 'county'];
```

Comment:
'Fixing' the address feels like it isn't something that fits with what we had talked about wrt the purpose of the ETL. Is it better suited to the `db-updater` layer? Although, with it being fairly fundamental to making the data a decent base to build on it might be ok?